kernel/data-pipeline/scenario/migration/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/migration/check/consistency/MigrationDataConsistencyChecker.java

```diff
@@ -77,12 +78,17 @@ public Map<String, DataConsistencyCheckResult> check(final DataConsistencyCalcul
         verifyPipelineDatabaseType(calculateAlgorithm, jobConfig.getTarget());
         SchemaTableName sourceTable = new SchemaTableName(new SchemaName(tableNameSchemaNameMapping.getSchemaName(jobConfig.getSourceTableName())), new TableName(jobConfig.getSourceTableName()));
         SchemaTableName targetTable = new SchemaTableName(new SchemaName(tableNameSchemaNameMapping.getSchemaName(jobConfig.getTargetTableName())), new TableName(jobConfig.getTargetTableName()));
+        progressContext.getTableNames().add(jobConfig.getSourceTableName());
```

Comment:
Why `progressContext.getTableNames().add(jobConfig.getSourceTableName());` is moved to here, it's separated with `progressContext.setRecordsCount(getRecordsCount());`, is it used in new added code block?

Reply:
Because the table name need be set before return if the unique key isn's exists

Reply:
OK